lib/Backends/CPU/CMakeLists.txt

```diff
@@ -79,3 +80,22 @@ target_link_libraries(CPUBackend
 add_dependencies(CPUBackend libjit_cpu_TARGET)
 
 set(linked_backends ${linked_backends} CPUBackend PARENT_SCOPE)
+
+# List the object files from "libobj_cpu" directory.
+FILE(GLOB objects "${CMAKE_CURRENT_LIST_DIR}/libobj_cpu/*.o")
```

Comment:
Could you clarify where this directory `libobj_cpu` with object files lives? In the source tree? If so, does it mean that all the *.o files are under source control? It would be a bit unusual. Is it a good idea to have those object files under source control inside Glow? Maybe they could live somewhere outside and some command-line parameter for CMake would provide a path to them, so that they can be found here and added to the bundle?

Reply:
The folder `libobj_cpu` currently lives in the source tree along with `libjit_cpu`.
Currently we have no objects in that folder (only a `Readme.txt`) so we can say this code serves as an example of how backends can serialize and bundle their own objects. But backends (either public or private) can choose a different folder where to store their objects, for example they can store them externally and add a CMake option to provide at Glow build-time the external folder which contains the objects. Each backend can specialize/modify this as they please.